config/sizes.json

```diff
@@ -34,5 +34,9 @@
     "handlebars-intl/dist/handlebars-intl.min.js": {
         "bytes": 7267,
         "kbs": 7.1
+    },
+    "ember-intl/dist/ember-intl.min.js": {
+        "bytes": 21362,
+        "kbs": 25
```

Comment:
I think in the `ember-intl` package is the right place to do this, that way the website can treat it simply as a pre-built dep like does for the other integrations.